IntelliTect.TestTools.TestFramework/IntelliTect.TestTools.TestFramework/ILogger.cs

```diff
@@ -2,8 +2,14 @@
 {
     public interface ILogger
     {
+        // Probably need to handle this differently
+        // Maybe a constructor sets the test case name?
```

Comment:
I agree with these comments. In general I don't like public setters like this since it puts the burden on the caller to use the object correctly. But for now it works.

Reply:
I think I had the realization last night that trying to create the logger via the DI service was limiting me from handling this "properly" (i.e. a constructor or some other more controlled method where I can hand in a few strings that set the needed properties.) I think for my next iteration I'll modify how the logger is added/overridden/created so I have the control of handing in constructor arguments. I'd rather get it addressed sooner rather than later since we're starting to use this as multiple clients.